File: rust/src/bin/0002.rs
use std::io::prelude::*;
use std::str::FromStr;

fn parse( s : String ) -> Vec<i32> {
    s.split_whitespace().map(|x| i32::from_str(x).unwrap()).collect()
}

fn ans( a:i32, b:i32 ) -> i32 {
    (a+b).to_string().len() as i32
}

fn main() {
    let reader = std::io::stdin();
    reader.lock().lines().for_each(|line|{
        let v= parse(line.unwrap());
        let n = ans(v[0],v[1]);
        println!("{:?}",n);
    });
}

File: python/0002.py
import sys

def ans(a):
    if a == 0:
        return 1
    else:
        r = 0
        while a > 0:
            r += 1
            a //= 10
        return r

for line in sys.stdin:
    [a,b] = list(map(int, line.split()))
    print(ans(a+b))
